packages/react-instantsearch-hooks/src/__tests__/useClearRefinements.test.tsx

```diff
@@ -0,0 +1,89 @@
+import { renderHook } from '@testing-library/react-hooks';
+import React from 'react';
+
+import { createInstantSearchTestWrapper } from '../../../../test/utils';
+import { useClearRefinements } from '../useClearRefinements';
+import { useRefinementList } from '../useRefinementList';
+
+import type { UseRefinementListProps } from '..';
```

Comment:
We can group it with this import line 6.

Reply:
it was copied from another one, maybe use use eslint to be consistent later?